sources/Context.cpp

```diff
@@ -149,11 +172,20 @@ Context::exportOutputs() {
   file::path outputDir(config_.outputDir());
 
   // Job
-  outputs::Job jobWriter(outputs::Job::JobDefinition(basename_, def_.dynawoLogLevel));
+  outputs::Job jobWriter(outputs::Job::JobDefinition(basename_, def_.dynawoLogLevel, config_));
```

Comment:
Now job writing is more complex than calling a writer, it should have a dedicated function. Same remark for the loop over the contingencies